### PR TITLE
feat: PyTorch LSTM service

### PR DESCRIPTION
**Pull request by jamesthatcher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # ML-python-pytorch-lstm-gpu-service
-Jenkins X quick start for training and deploying LSTM Neural Network Classification using the PyTorch library
+
+Quickstart project for executing an IMDB classifier using the PyTorch framework on a GPU.
+
+Running `pip install requirements.txt` and then python `app.py` will start the app on localhost where the user can send POST requests to perform inference.
+
```
Comment (mxdillon) [inline README.md]: Can you include a bit of info and example of what format / type of POST request needs to be sent (ie json with text ; what the endpoint is ...)

File: jenkins-x.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+buildPack: ml-python-service
```
Comment (mxdillon) [inline jenkins-x.yml]: Think this has to have `-gpu` on the end - check the other GPU services

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+jsonpickle==1.3
+future==0.17.1
+protobuf==3.7.1
+falcon==1.4.1
+numpy==1.16.2
+pytest==5.3.2
+onnx==1.6.0
+pytorch==1.4
+torchvision==0.5.0
+onnxruntime-gpu==1.2.0
```
Comment (mxdillon) [inline requirements.txt]: alphabetical order

File: service.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+start = time.time()
+
+# Load the ONNX model
+model = onnx.load("model.onnx")
+
+# Check that the IR is well formed
+onnx.checker.check_model(model)
+
+so = ort.SessionOptions()
+so.graph_optimization_level = ort.GraphOptimizationLevel.ORT_ENABLE_ALL
```
Comment (mxdillon) [inline service.py]: what does this do?

File: service.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+    def on_post(self, req, resp):
+        req.client_accepts('text/plain')
+        doc = str(req.stream.read())
+        processed = TEXT.process(([TEXT.preprocess(doc)]))
+        padded = nn.ConstantPad1d((0, input_len - processed.shape[1]), 0)(processed)
+        result = sess.run([output_name], {input_name: padded.numpy()})
+
+        # Subset results
+        print(result)
```
Comment (mxdillon) [inline service.py]: do you need to print the results here? - remove for production?

File: service.py (hunk 4 above)
Comment (jamesthatcher) [reply service.py:21]: There is a good explanation in the ONNX docs [here](https://github.com/microsoft/onnxruntime/blob/master/docs/ONNX_Runtime_Graph_Optimizations.md)

**Pull request merged**